src/store/transferSlice.ts

```diff
@@ -308,7 +298,32 @@ export const transferSlice = createSlice({
     ) => {
       state.acalaRelayerInfo = receiveDataWrapper(action.payload);
     },
-  },
+    setNeonRelayerInfo: (
+      state,
+      action: PayloadAction<AcalaRelayerInfo | undefined>
+    ) => {
+      state.acalaRelayerInfo = action.payload
```

Comment:
Can we add a new `neonRelayerInfo` to the `state`

Reply:
Fixed it